smithy-openapi/src/main/java/software/amazon/smithy/openapi/model/OperationObject.java

```diff
@@ -104,8 +108,8 @@ public boolean isDeprecated() {
         return deprecated;
     }
 
-    public List<Map<String, List<String>>> getSecurity() {
-        return security;
+    public Optional<List<Map<String, List<String>>>> getSecurity() {
```

Comment:
You could make this method always return a List by making the list of security schemes nullable. You'd return the value if present, otherwise an empty list.

Reply:
The problem then is from the outside you don't know whether the list was deliberately set to an empty list or just not set, unless you go out of you way to look at the generated node.